### ISSUE TITLE
How to add a custom Estimator

### ISSUE THREAD

**Issue by xieweilun** [imranq2/SparkPipelineFramework#94]
Hello
    I saw adding the Estimator class on the README.
    However, the FrameworkPipeline class does not perform any processing on the self.estimator.
    Is this a feature that is not currently supported?
    Looking forward to your reply. Thank you

**Comment by imranq2** [comment]
Yes, it has not been added yet.  feel free to submit a PR otherwise we'll probably get to it in a few months.
